examples/lifelong_learning/atcii/train.py

```diff
@@ -28,13 +28,18 @@ def main():
     train_data.parse(train_dataset_url, label=DATACONF["LABEL"])
     attribute = json.dumps({"attribute": DATACONF["ATTRIBUTES"]})
     early_stopping_rounds = int(
-        Context.get_parameters(
-            "early_stopping_rounds", 100))
+        Context.get_parameters("early_stopping_rounds", 100)
+    )
```

Comment:
change `100` to a constant

Reply:
In fact, this is a developer-specified hyperparameter and should not be obtained from lib.